dll/shellext/zipfldr/zipfldr.cpp

```diff
@@ -174,16 +174,17 @@ STDAPI DllUnregisterServer()
 
 EXTERN_C
 BOOL WINAPI
-RouteTheCall(
-    IN HWND hWndOwner,
-    IN HINSTANCE hInstance,
-    IN PCWSTR lpStringArg,
-    IN INT Show)
+RouteTheCallW(IN HWND hWndOwner, IN HINSTANCE hInstance, IN PCWSTR lpStringArg, IN INT Show)
```

Comment:
I think it would be better to preserve function prototype formatting:
```suggestion
RouteTheCallW(
    IN HWND hWndOwner,
    IN HINSTANCE hInstance,
    IN PCWSTR lpStringArg,
    IN INT Show)
```

Reply:
It is the only function in that file that used this format, all the others are one line.

And it is a standard rundll32 function, you should know by heart what the parameters are.

Reply:
I just want the diff to be more clear about the fix, keeping formatting changes in a separate commit, if you want it to be consistent throughout the file.